clasp/clasp_facade.h

```diff
@@ -449,6 +519,8 @@ class ClaspFacade : public ModelHandler {
 	BuilderPtr   builder_;
 	SummaryPtr   accu_;
 	StatsPtr     stats_; // statistics: only if requested
+	std::vector<UserdefinedStatsCallback> userStatsCbs_;
```

Comment:
Consider using a single vector storing pairs of function pointer and corresponding data.